Artie integration (ECOINT-13) (#2513)
DataDog/integrations-extras: artie/README.md

# Artie Integration

## Overview

> [Artie][1] is a real-time database replication product for reliably and consistently syncing data from source databases to data warehouses.

Reviewer: ```suggestion
[Artie][1] is a real-time database replication product for syncing data from source databases to data warehouses.
```
Sorry, I need to remove marketing language.

With this integration, you can receive metrics about your Artie deployments directly to your Datadog account in order to monitor the health of your data pipelines. Metrics include number of rows processed over time, ingestion lag, flush time, and replication slot size.

## Setup

### Installation

> This integration is only available for existing Artie accounts. If you're not using Artie yet and would like to start a trial, contact us at hi@artie.com.

Reviewer: ```suggestion
This integration is only available for existing Artie accounts. If you're not using Artie yet and would like to start a trial, email hi@artie.com.
```

1. Click **Connect Accounts** on the Artie integration tile to connect Datadog with Artie.
2. Log into Artie if you aren't logged in yet.
3. Review the Datadog permissions that will be granted to Artie and click **Authorize**.

### Configuration

The integration will automatically send metrics to Datadog; no further configuration is needed.

### Validation

> The included dashboard should start to show data within 5-10 minutes of connecting the integration, as long as you have data flowing in your Artie deployments.

Reviewer: ```suggestion
The included dashboard start to show data 5-10 minutes after you connect the integration, as long as you have data flowing in your Artie deployments.
```
